drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/base/CheckStorage.java

```diff
@@ -51,25 +50,16 @@ public CheckStorage( final CheckFactory checkFactory ) {
 
         ruleInspectorChecks.addAllValues( ruleInspector,
                                           checks );
-        return checks;
     }
 
-    private HashSet<Check> makePairRowChecks( final RuleInspector ruleInspector,
-                                              final Collection<RuleInspector> all ) {
-        final HashSet<Check> checks = new HashSet<>();
+    private void makePairRowChecks( final RuleInspector ruleInspector,
+                                    final Collection<RuleInspector> all ) {
 
         for ( final RuleInspector other : all ) {
             if ( !ruleInspector.equals( other ) ) {
-                final Optional<PairCheckBundle> pairCheckList = checkFactory.makePairRowCheck( ruleInspector,
-                                                                                               other );
-                if ( pairCheckList.isPresent() ) {
-                    checks.add( pairCheckList.get() );
-                    pairCheckStorage.add( pairCheckList.get() );
-                }
+                checkFactory.makePairRowCheck( ruleInspector, other ).ifPresent( pairCheckStorage::add );
```

Comment:
@jomarko checks Set was only used as return value, but then the returned value was not used by the invoker of that method, so I just make the method to return void.